src/assets/js/app.js

```diff
@@ -1,9 +1,28 @@
 document.addEventListener("DOMContentLoaded", function(event) { 
-    if(navigator.userAgent.toLowerCase().indexOf("android") > -1){
+    const originalUrl = sessionStorage.getItem('originalUrl');
+    sessionStorage.removeItem('originalUrl');
+    if (originalUrl) {
+        const originalUrlElement = document.getElementById("originalUrl");
+        originalUrlElement.classList.remove("d-none");
```

Comment:
![2021-05-05 11_34_21-Window](https://user-images.githubusercontent.com/4686874/117122197-ea71f000-ad95-11eb-8ba4-4991924f479b.png)

There is a visible printed originalUrl o the website when opening the main website and being redirected to the languageCode `/en` or `/de`